compiler/optimizer/Dominators.cpp

```diff
@@ -459,14 +457,15 @@ void TR_PostDominators::findControlDependents()
    int32_t nextNodeNumber = _cfg->getNextNodeNumber();
    int32_t i;
 
+   _directControlDependents = (TR_BitVector**)_region.allocate(nextNodeNumber*sizeof(TR_BitVector *));
```

Comment:
Was double indirection necessary or cleaner here?  At first glance this looks like it could 'just' be a dynamically allocated array of values, but that it might also be more trouble than it's worth.